apps/astarte_appengine_api/mix.exs

```diff
@@ -33,7 +33,9 @@ defmodule Astarte.AppEngine.API.Mixfile do
         "coveralls.post": :test,
         "coveralls.html": :test
       ],
-      dialyzer_cache_directory: dialyzer_cache_directory(Mix.env()),
+      dialyzer: [
+        plt_core_path: "./#{dialyzer_cache_directory(Mix.env())}"
```

Comment:
Doesn't this work without prefixing it with `./`? I would assume paths are relative by default. If it doesn't work I'd fix the `dialyzer_cache_directory` function.

Reply:
`plt_core_path` is absolute, I changed `dialyzer_cache_directory`